WooCommerce/Classes/POS/Presentation/Item Selector/SimpleProductCardView.swift

```diff
@@ -57,6 +66,6 @@ private extension SimpleProductCardView {
                                                     price: "3.00",
                                                     manageStock: false,
                                                     stockQuantity: nil,
-                                                    stockStatusKey: ""))
+                                                    stockStatusKey: "In stock"))
```

Comment:
nit: I think the key for in-stock enum is `instock`?

Reply:
You're right! https://github.com/woocommerce/woocommerce-ios/pull/15670/commits/ca8618d89b1c962ee075ce803e7ca6bc8bec2f95